### ISSUE TITLE
Confirm the namelist for the MPAS init_atmosphere

### ISSUE THREAD

**Issue by guoqing-noaa** [rrfs2/rrfs-workflow#82]
This is from Tanya in an email thread, the ` data_source section` section of `namelist.init_atmosphere` should be as follows:   

```
&data_sources
    config_geog_data_path = '/lfs4/NAGAPE/hpc-wof1/ywang/MPAS/WPS_GEOG/'
    config_met_prefix = 'RAP'
    config_sfc_prefix = 'SST'
    config_fg_interval = 10800
    config_landuse_data = 'MODIFIED_IGBP_MODIS_NOAH_15s'
    config_topo_data = 'GMTED2010'
    config_vegfrac_data = 'MODIS'
    config_albedo_data = 'MODIS'
    config_maxsnowalbedo_data = 'MODIS'
    config_supersample_factor = 3
    config_30s_supersample_factor = 1
    config_use_spechumd = true
    config_soilcat_data = 'BNU'
/
```
Note the following two lines have been updated recently:   
    **config_landuse_data = 'MODIFIED_IGBP_MODIS_NOAH_15s'**
    **config_soilcat_data = 'BNU'**  
    
Per Tanya:
```
These are the sources that are actually used to produce a static file.
When we use the wrong data sources in the namelist for init/ctl, we replace the global attributes
 with the wrong names (BNU -> STATSGO, MODIFIED_IGBP_MODIS_NOAH_15s -> MODIFIED_IGBP_MODIS_NOAH), 
but the data is still BNU and MODIFIED_IGBP_MODIS_NOAH_15s. 
It is confusing, but does not have an impact on model performance.
```
Per @daviddowellNOAA :
···
The new namelists that are being used with new physics in MPAS_physics_dev1 differ in some significant ways
 from the old namelists that are used for MPAS runs with the old physics.  The new namelists have to be different 
if we want to activate some of the new features in MYNN, Thompson microphysics, etc.
···
Here is the presentation prepared by David summarizing the main differences in the old and new namelists.
https://docs.google.com/presentation/d/1u-DSMEwFd7n-wAFJrEJQsgVBo7BdiDJHm2JuYL11cNU/edit#slide=id.g278550fb798_0_4

**Comment by daviddowellNOAA** [comment]
I agree with Tanya's suggested changes to namelist.init_atmosphere, for the reasons she stated.  These changes were made for real-time runs on July 25.  The change log for mpas_physics_dev1 (aka. hrrv5) is found here on Jet:
/misc/whome/role.wrfruc/HRRRv5/README

**Comment by guoqing-noaa** [comment]
Thanks, @daviddowellNOAA 
Do we need to update the grid.nc, staic.nc, and (MPASSIT_)init.nc as well?

**Comment by daviddowellNOAA** [comment]
The existing grid.nc and static.nc are good.  We used the correct namelist.init_atmosphere to generate static.nc.

All files created during the MPAS initialization (e.g., init.nc) and forecast could be affected by the recent bug fix to the real-time version of namelist.init_atmosphere.

**Comment by guoqing-noaa** [comment]
Thanks! @daviddowellNOAA
